app/src/main/res/layout/activity_main.xml

```diff
@@ -7,65 +7,63 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
-    <com.google.android.material.appbar.AppBarLayout
+<LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+    <TableRow
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
-        app:layout_constraintTop_toTopOf="parent">
+        android:background="@color/colorGrayTabBackground"
+        android:gravity="center">
 
-        <TableRow
+        <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:background="@color/colorGrayTabBackground"
-            android:gravity="center">
-
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:orientation="vertical">
+            android:layout_height="match_parent"
+            android:orientation="vertical">
 
-                <ImageView
-                    android:id="@+id/imageViewLogo"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:contentDescription="@string/app_name"
-                    android:paddingLeft="@dimen/logoPaddingLeft"
-                    android:paddingTop="@dimen/logoPaddingTop"
-                    app:srcCompat="@drawable/ic_pizzap" />
+            <ImageView
```

Comment:
sould put android:layoutDirection="ltr" so it won't jump to the other side in hebrew

Reply:
thanks